src/Microsoft.Extensions.Http/Logging/EventIds.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Http.Logging
+{
+    internal static class EventIds
```

Comment:
These are only used in the `Log` static method. Why not just have them be constants there? Also, use `nameof`?

Reply:
These eventIds are shared across multiple classes writing to the same logger category.

Reply:
I don't want to use nameof here. These names should **not** get refactored.